web/src/components/people/ApprovedResearcherImport.tsx

```diff
@@ -0,0 +1,43 @@
+import Button from "@/components/ui/Button";
+import { postPeopleApprovedResearchersImportCsv } from "@/openapi";
+import { useRef, useState } from "react";
+
+export default function ApprovedResearcherImport() {
+  const inputRef = useRef<HTMLInputElement>(null);
+  const [buttonText, setButtonText] = useState("Import approved researchers");
+  const [buttonDisabled, setButtonDisabled] = useState(false);
+
+  async function handleFileUpload(e: React.ChangeEvent<HTMLInputElement>) {
+    const files = e.target.files;
+    if (!files) return;
+
+    const response = await postPeopleApprovedResearchersImportCsv({
+      body: files[0],
+    });
+    if (response.error) {
+      console.log(response.error);
+      setButtonText("Failed");
+    } else {
+      setButtonText("Imported ✔");
+    }
+    setButtonDisabled(true);
+  }
+
+  function handleSumbit(e: React.MouseEvent<HTMLButtonElement>) {
```

Comment:
I think this could actually be a `FormEvent` like what we use for [setting chosen name](https://github.com/ucl-arc-tre/portal/blob/4a333bc8df843543069b90a54b302c814d5786fb/web/src/components/profile/ProfileChosenName.tsx#L32C56-L32C56)